templates/default/services.conf.erb

```diff
@@ -115,9 +115,20 @@ filter {
     dissect {
       mapping => {"message" => "%{templogdate} %{+templogdate} %{priority} %{logger_name}: %{log_message}"}
     }
-    json {
-      source => "log_message"
-      skip_on_invalid_json => true
+		clone {
+			clones => ["onlinefs_service_log"]
+		}
+		if [type] != "onlinefs_service_log" {
```

Comment:
maybe we could generify tags:
`onlinefs_service_log` -> `service_log` 
`onlinefs_project_log` -> `project_log`